packages/core/src/indexing/local-index-api.ts

```diff
@@ -26,6 +26,14 @@ export class LocalIndexApi implements IndexApi {
     private readonly logger: DiagnosticsLogger,
     networkName: Networks
   ) {
+    // the '?' is here, because we're passing `unknown` as the indexingConfig in tests. refactor the tests maybe?
+    if (indexingConfig?.models !== undefined) {
+      // TODO: Update the log once admin API is implemented
+      logger.warn(`
+      Passing stream IDs to be indexed via indexing config is deprecated and will soon be removed as an option.
```

Comment:
```suggestion
      Passing the list of Model StreamIDs to be indexed the config file's 'indexing.models' field is deprecated and will soon be removed as an option.
```

Reply:
Want to clarify we're talking about indexing models, not specific streamids, and call out the field name of the field in the config file that is now deprecated

Reply:
Awesome, thanks for this, @stbrody !